worker/exec/konnector_test.go

```diff
@@ -361,5 +367,45 @@ echo "{\"type\": \"toto\", \"message\": \"COZY_URL=${COZY_URL}\"}"
 		assert.Contains(t, dir.CozyMetadata.CreatedOn, inst.Domain)
 		assert.Len(t, dir.CozyMetadata.UpdatedByApps, 1)
 		assert.Equal(t, dir.CozyMetadata.SourceAccount, acc.ID())
+
+		// Folder is created from Konnector name and Account name
+		wg.Add(1)
+		acc.DefaultFolderPath = ""
+		acc.Name = "account-1"
+		require.NoError(t, couchdb.GetDoc(inst, consts.Accounts, acc.ID(), acc))
+
+		msg, err = job.NewMessage(map[string]interface{}{
+			"konnector":      "my-konnector-1",
+			"folder_to_save": "id-of-a-deleted-folder",
+			"account":        acc.ID(),
+		})
+		require.NoError(t, err)
+
+		j = job.NewJob(inst, &job.JobRequest{
+			Message:    msg,
+			WorkerType: "konnector",
+		})
+
+		config.GetConfig().Konnectors.Cmd = tmpScript
```

Comment:
Maybe we could add a defer to reset this configuration value to its old value at the end of this test?

Reply:
Is that change really saved?

Reply:
It changes a global variable of the config package, it is not persisted on disk.